Frontend/src/app/dashboard/components/customers/customers.component.html

```diff
@@ -14,5 +14,9 @@ <h3>{{title}}</h3>
       <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
       <tr mat-row *matRowDef="let row; columns: displayedColumns;"></tr>
     </table>
+
+    <mat-paginator #paginator [pageSize]="pageEvent.pageSize" [pageSizeOptions]="[5, 10]" [length]="pageEvent.length"
```

Comment:
pageSizeOption też dobrze byłoby zdefiniowac jako zmienna w pliku .ts

Reply:
Wyszedłem z założenia, że skoro ta zmienna nie jest dynamicznie połączona z danymi tylko ustawiana raz, to może być w html'u, ale faktycznie lepiej to wygląda gdy wszystko jest przeniesione do ts'a